app/upgrades/v19/upgrades.go

```diff
@@ -36,10 +42,19 @@ func CreateUpgradeHandler(
 		ctxCache, writeFn := ctx.CacheContext()
 		if err := RemoveOutpostsFromEvmParams(ctxCache, ek); err == nil {
```

Comment:
This will only be needed for testnet no? Cause on mainnet we already emptied the ActivePrecompiles slice